.gitignore

```diff
@@ -1,5 +1,6 @@
-node_modules
-bower_components
-npm-debug.log
+bower_components/
+node_modules/
+dist/
 .DS_Store
```

Comment:
Always feel this should live in a global config on dev machines rather than being a project concern. Belt and braces though.

Reply:
Agreed! But yes, belt and braces